tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/asynchronous/AsyncClient.java

```diff
@@ -41,26 +41,13 @@
 public class AsyncClient {
 
     /**
-     * service 1 second in normal operation.
-     *
-     * @return the result as a Future
-     * @throws InterruptedException the interrupted exception
+     * Service an operation until waitCondition is completed or 1 second timeout.
+     * @param waitCondition Execution of this method will delay until the condition is finished
+     * @return the result as a Future.
```

Comment:
Same here, javadoc is incorrect. (It wasn't correct before either.)

Reply:
done